src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -284,14 +273,12 @@ bool emberAfGroupsClusterGetGroupMembershipCallback(app::CommandHandler * comman
 
     {
         GroupDataProvider::EndpointIterator * iter = nullptr;
-        CHIP_ERROR err                             = CHIP_NO_ERROR;
 
         iter = provider->IterateEndpoints(fabricIndex);
         VerifyOrExit(nullptr != iter, status = EMBER_ZCL_STATUS_FAILURE);
 
-        err = commandObj->AddResponseData(commandPath, GroupMembershipResponse(commandData, commandPath.mEndpointId, iter));
+        commandObj->AddResponse(commandPath, GroupMembershipResponse(commandData, commandPath.mEndpointId, iter));
         iter->Release();
-        status = (CHIP_NO_ERROR == err) ? EMBER_ZCL_STATUS_SUCCESS : EMBER_ZCL_STATUS_FAILURE;
```

Comment:
This still needs to set status to EMBER_ZCL_STATUS_SUCCESS, no?

Reply:
Yes, we should set status here.